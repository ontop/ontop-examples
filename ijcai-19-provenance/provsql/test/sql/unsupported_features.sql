\set ECHO none
SET search_path TO public, provsql;

SELECT provenance();

SELECT * FROM (VALUES (1)) t, personnel;

WITH Q(a) AS (SELECT 1) SELECT * FROM Q, personnel;

SELECT COUNT(*) FROM personnel;

SELECT * FROM personnel WHERE city IN (SELECT city FROM personnel);

SELECT DISTINCT ON (city) * FROM personnel;

SELECT DISTINCT 1 FROM personnel GROUP BY city;

SELECT * FROM personnel INTERSECT SELECT * FROM personnel;
