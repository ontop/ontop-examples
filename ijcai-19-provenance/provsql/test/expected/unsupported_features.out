\set ECHO none
ERROR:  provenance() called on a table without provenance
ERROR:  FROM clause unsupported by provsql
ERROR:  FROM clause unsupported by provsql
ERROR:  Aggregation not supported by provsql
ERROR:  Subqueries in WHERE clause not supported by provsql
ERROR:  DISTINCT ON not supported by provsql
ERROR:  Inconsistent DISTINCT and GROUP BY clauses not supported by provsql
ERROR:  Set operations other than UNION and EXCEPT not supported by provsql
